src/cpp/editor/hero_edit.cpp

```diff
@@ -1,17 +1,39 @@
-//#include "base.h"
-
-#include "adventure/map.h"
+#include "hero_edit.h"
 
+#include "artifacts.h"
 #include "editor.h"
 #include "gui/msg.h"
 
 extern HeroExtra gEditedHeroExtra;
 
-extern int __fastcall EditHeroHandler_orig(tag_message&);
-extern void __stdcall FillInHeroEdit(HeroExtra*);
-
 #define NUM_HERO_PORTRAITS 72
 
+namespace {
+  bool fillInArtifacts = true;
```

Comment:
I see. So, the original code was actually in EditHero, but you moved it to FillInHeroEdit because it was easier than moving EditHero into the Ironfist codebase? Makes sense.